app/models/store.rb

```diff
@@ -22,6 +22,13 @@ def active_items
     items.where(condition: "active")
   end
 
+  def update_status(status)
```

Comment:
This method was created to both update the store and update the role of the user that created the store.

This will need to be updated to be more flexible and take in different types of updates.